### ISSUE TITLE
Initial Setup fails

### ISSUE THREAD

**Issue by waja** [hoellen/dockerfiles#33]
Hi @hoellen,

after creating a new instance nextcloud isn't configured.

```
nextcloud_1            | Updating permissions in /data...
nextcloud_1            | Updating permissions in /config...
nextcloud_1            | Updating permissions in /apps2...
nextcloud_1            | Updating permissions in /var/log...
nextcloud_1            | Updating permissions in /php...
nextcloud_1            | Updating permissions in /nginx...
nextcloud_1            | Updating permissions in /tmp...
nextcloud_1            | Updating permissions in /etc/s6.d...
nextcloud_1            | Done updating permissions.
nextcloud_1            | Starting automatic configuration...
nextcloud_1            | Automatic configuration finished.
nextcloud-db_1         | 2020-01-27 10:38:47 10 [Warning] 'proxies_priv' entry '@% root@f67885bdea72' ignored in --skip-name-resolve mode.
nextcloud-db_1         | 2020-01-27 10:38:47+01:00 [Note] [Entrypoint]: Creating database nextcloud
nextcloud-db_1         | 2020-01-27 10:38:47+01:00 [Note] [Entrypoint]: Creating user nextcloud
nextcloud-db_1         | 2020-01-27 10:38:47+01:00 [Note] [Entrypoint]: Giving user nextcloud access to schema nextcloud
nextcloud_1            | Nextcloud is not installed - only a limited number of commands are available
nextcloud-db_1         | 2020-01-27 10:38:47 14 [Warning] 'proxies_priv' entry '@% root@f67885bdea72' ignored in --skip-name-resolve mode.
nextcloud-db_1         | 
nextcloud-db_1         | 2020-01-27 10:38:47+01:00 [Note] [Entrypoint]: Stopping temporary server
nextcloud_1            | 
nextcloud_1            |                                          
nextcloud_1            |   Command "app:disable" is not defined.  
nextcloud_1            |                                          
nextcloud_1            |   Did you mean this?                     
nextcloud_1            |       app:check-code                     
nextcloud_1            |                                          
nextcloud_1            | 
```
Running `/usr/local/bin/setup.sh` again inside the container:
```
/nextcloud # /usr/local/bin/setup.sh 
Starting automatic configuration...
Automatic configuration finished.
Nextcloud is not installed - only a limited number of commands are available

                                                                                                                        
 Command "app:disable" is not defined.                                                                                  
                                                                                                                        

 Do you want to run "app:check-code" instead?  (yes/no) [no]:
 > yes

Analysing /nextcloud/apps/firstrunwizard/templates/wizard.php
 1 errors
    line   34: OC_User - Static method of private class must not be called
Analysing /nextcloud/apps/firstrunwizard/lib/AppInfo/Application.php
 2 errors
    line   58: OC_Util - Static method of private class must not be called
    line   76: OC_Util - Static method of private class must not be called
App is not compliant
```
Using `b8f4bae0e6cd` and `a21474b1fc1e` as image works as expected.

**Comment by hoellen** [comment]
This sounds a lot like #15. I still can't reproduce it. 
I use tag `18.0` (image `b3ff3e88bac5`) and the following testing environment variables:
```
sudo docker run --rm --name nextcloud \
       -e UID=1000 -e GID=1000 \
       -e UPLOAD_MAX_SIZE=10G \
       -e APC_SHM_SIZE=128M \
       -e OPCACHE_MEM_SIZE=128 \
       -e CRON_PERIOD=15m \
       -e ADMIN_USER=mrrobot \
       -e ADMIN_PASSWORD=supercomplicatedpassword \
       -e DOMAIN=cloud.example.com \
       -p 80:8888 \
hoellen/nextcloud:18.0
```
Automatic configuration works fine.
What tag you are using? Which values have your environment variables? Are you using something special, like NFS volumes?

**Comment by waja** [comment]
Hi there,

I'm running a couple of instances since some time. Today I thought it's time to deploy another one (by a script like every time, for more see https://github.com/waja/nextcloud-docker-multiinstance).
It doesn't matter if I use the actual tags `18.0` or `17.0`, both fail to setup. Using the images published 7 days ago (`b8f4bae0e6cd` and `a21474b1fc1e`) works well.

My environment:
```
    environment:
      ADMIN_PASSWORD: s3cr3t
      ADMIN_USER: admin
      APC_SHM_SIZE: 128M
      CRON_PERIOD: 15m
      DB_HOST: nextcloud-db
      DB_NAME: nextcloud
      DB_PASSWORD: zEtzqeahaha
      DB_TYPE: mysql
      DB_USER: nextcloud
      DOMAIN: nextcloud.test.org
      GID: '1000'
      OPCACHE_MEM_SIZE: '128'
      TZ: Europe/Berlin
      UID: '1000'
      UPLOAD_MAX_SIZE: 10G
    image: b8f4bae0e6cd
```

Anyway ... running any existing instance with recent `18.0` or `17.0` seems not to cause any issues so far.

Maybe it is caused by the update of PHP or nginx?

**Comment by waja** [comment]
Hi,

I think you can close this.
On an other system I was able to deploy a new instance without hassle.

Sorry for the noise and thanks, Jan.
